CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 
 ## 1.6.1
 * Add webhooks into travis.yml to notifiy galaxy about new releases.
+* Replace kamaln7.swapfile with geerlingguy.swap requeriment role.
```

Comment:
@speedlight I think we need to update the CHANGELOG since we are currently at version v1.6.2